src/Entity/ServiceProvider.php

```diff
@@ -106,6 +106,20 @@ class ServiceProvider extends AbstractRole
     public $requestedAttributes;
 
     /**
+     * @var null|string
+     *
+     * @ORM\Column(name="url_en", type="string")
+     */
+    public $urlEn;
```

Comment:
As @rjkip said, supportUrlNl would've been better. The 1:1 mapping to janus names is for historical reasons only and should certainly not get in the way of clearer names.
But this is okay too.